server/src/main/scala/za/co/absa/atum/server/api/http/Endpoints.scala

```diff
@@ -54,6 +54,15 @@ trait Endpoints extends BaseEndpoints {
       .out(jsonBody[AdditionalDataSubmitDTO])
   }
 
+  protected val getPartitioningCheckpointsEndpoint
```

Comment:
We will have endpoints of different versions alongside one another therefore I would suggest a suffix V2 `getPartitioningCheckpointsEndpointV2`.